templates/kubernetes/base/deployment.yml

```diff
@@ -71,3 +71,23 @@ spec:
   maxReplicas: 2
   minReplicas: 1
   targetCPUUtilizationPercentage: 60
+---
```

Comment:
Could you move this into its own file? The deployment manifest should be for anything related to the k8s `Deployment`. The job runs independently and most likely will use a different image, execute a different binary, etc. (remember to add the new file to `kustomization.yml`